README.md

```diff
@@ -4,4 +4,10 @@
 
 Java SDK for Nervos [CKB](https://github.com/nervosnetwork/ckb).
 
+You can generate the jar and import manually.
+```
+git clone https://github.com/nervosnetwork/ckb-sdk-java.git
+gradle shadowJar
+```
+
 Note: this is at very early stage, treat it as a PoC implementation.
```

Comment:
I guess we can remove this line now that SDKs will keep update with CKB core.

Reply:
OK, I remove it